pypdf/constants.py

```diff
@@ -149,8 +149,11 @@ class FileSpecificationDictionaryEntries:
 
     Type = "/Type"
     FS = "/FS"  # The name of the file system to be used to interpret this file specification
-    F = "/F"  # A file specification string of the form described in Section 3.10.1
+    F = "/F"  # A file specification string of the file as described in Section 3.10.1
+    UF = "/UF"  # A unicode string of the file as described in Section 3.10.1
     EF = "/EF"  # dictionary, containing a subset of the keys F , UF , DOS , Mac , and Unix
+    RF = "/RF"  # dictionary, containing arrays of /EmbeddedFile
+    DESC = "/Desc"  # description of the file as de
```

Comment:
Those could be in an individual PR - that would be easy to review/quick to merge :-)